data/recommendation/notify.js

```diff
@@ -2,8 +2,7 @@ const yo = require('yo-yo');
 
 class Notify {
   start() {
-    this.target = this.createBase();
-    document.body.appendChild(this.target);
+    this.target = document.body.appendChild(yo`<div></div>`);
```

Comment:
Does this work? I thought we had issues with needing an id to properly update the ui? Also, I think you can use a self closing tag, like ``yo`<div />` ``.